src/plots/cartesian/graph_interact.js

```diff
@@ -309,6 +309,13 @@ fx.unhover = function (gd, evt, subplot) {
 // The actual implementation is here:
 
 function hover(gd, evt, subplot){
+    if(gd._fullLayout._hasPie){
```

Comment:
The contents of `[evt]` mimic the same "format" as other traces hover events `points` value (as much as we can with a pie chart) but `xvals/yvals` and `xaxes/yaxes` are out of context here. The `xvals/yvals` are the mouse hover position relative to the axes scaling and values, so without any axes, these are meaningless.